metasrv/src/meta_service/raftmeta.rs

```diff
@@ -271,7 +273,7 @@ impl MetaNode {
 
         if !is_open {
             if let Some(_addrs) = init_cluster {
-                mn.init_cluster(config.raft_api_addr()).await?;
+                mn.init_cluster(config.raft_api_addr().await?);
```

Comment:
Do not we need `await` for `mn.init_cluster()`?

Reply:
> Do not we need `await` for `mn.init_cluster()`?

For the rpc is `async,` this `async` changed the method of raft_api_addr attribute, from sync to async.
So, it should this await keyword to `wait` the Future done.

Reply:
> Do not we need `await` for `mn.init_cluster()`?

Let me check it again.

Reply:
> Do not we need `await` for `mn.init_cluster()`?

Indeed, sure.